inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -1978,6 +1979,15 @@ void convertFunctionToICNNNetwork(const std::shared_ptr<const ::ngraph::Function
             cnnLayer->params[ExecGraphInfoSerialization::ORIGINAL_NAMES] = originalNames;
         }
 
+        if (auto subgraph = ::ngraph::as_type_ptr<ngraph::snippets::op::Subgraph>(layer)) {
```

Comment:
Hmmm, can we use `addSpecificCreator` to set originalLayersNames?

Reply:
As discussed offline this code is used for execution graphs like original names after handcrafted fusion in CPU plugin so can be kept here